src/test/java/edu/usf/cutr/gtfsrtvalidator/test/feeds/combined/TripUpdateVehiclePositionTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.usf.cutr.gtfsrtvalidator.test.feeds.combined;
+
+import edu.usf.cutr.gtfsrtvalidator.FeedMessageTest;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.combined.VehicleTripDescriptorValidator;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+import org.junit.Test;
+
+
+public class TripUpdateVehiclePositionTest extends FeedMessageTest{
+    
+    public TripUpdateVehiclePositionTest() throws Exception{}
+    
+    @Test
+    public void testTripAndVehicleDescriptorValidation() {
+         VehicleTripDescriptorValidator vehicleAndTripDescriptorValidator = new VehicleTripDescriptorValidator();
+        
+        vehiclePosition.setTrip(tripDescriptor);
+        feedEntity.setVehicle(vehiclePosition);
+        feedMessage.setEntity(0, feedEntity);
+        
+        errors = vehicleAndTripDescriptorValidator.validate(gtfsData, feedMessage.build());
+        assertEquals(0, errors.getOccurrenceList().size());
+        
+        /*If trip_id's and vehicle_id's in TripUpdate and VehiclePosition are not equal, validator should return an error.
+        That is it fails the validation and passes the assertion.
+        */        
+        //set trip_id = 100 in VehiclePosition i.e., not equal to trip_id = 1.1 in TripUpdate
+        tripDescriptor.setTripId("100");
+        vehiclePosition.setTrip(tripDescriptor);
+        //set vehicle_id = 1 in VehiclePosition i.e., not equal to vehicle_id = "" in TripUpdate
```

Comment:
Same as above - for tests I'd prefer to explicitly set the values that we're testing against in the test itself.  I believe the `vehicle_id = ""` is set in the super class as part of initializiation, but we should set a value here within the test method to keep the test parameters as encapsulated as possible.